src/main.rs

```diff
@@ -746,7 +819,7 @@ fn backtrace(
                     .transpose()?
                     .unwrap_or(Cow::Borrowed("???"));
 
-                println!("{:>4}: {}", frame_index, name);
+                backtrace_display_str = format!("{}{:>4}: {}\n", backtrace_display_str, frame_index, name);
```

Comment:
What do you think of doing this? This makes it faster clear that we are extending the current backtrace for me.
```suggestion
                backtrace_display_str.push_str(format!("{:>4}: {}\n", frame_index, name));
```

Reply:
There are some similar lines below

Reply:
ahhh perfect that was the non-hacky solution I was looking for, thank you